lib/vrl/stdlib/src/parse_ruby_hash.rs

```diff
@@ -109,36 +140,57 @@ fn parse_nil<'a, E: ParseError<&'a str>>(input: &'a str) -> IResult<&'a str, Val
 fn parse_bytes<'a, E: HashParseError<&'a str>>(input: &'a str) -> IResult<&'a str, Bytes, E> {
     context(
         "bytes",
-        map(
-            preceded(char('\"'), cut(terminated(parse_str::<'a, E>, char('\"')))),
-            |value| Bytes::copy_from_slice(value.as_bytes()),
-        ),
+        map(alt((parse_str('"'), parse_str('\''))), |value| {
+            Bytes::copy_from_slice(value.as_bytes())
+        }),
     )(input)
 }
 
+fn parse_simple_key<T, E: ParseError<T>>(input: T) -> IResult<T, T, E>
```

Comment:
I'd rename this to `parse_symbol_key` to match Ruby's terminology: https://ruby-doc.org/core-2.5.1/Hash.html